services/director-v2/tests/integration/shared_comp_utils.py

```diff
@@ -75,8 +76,8 @@ async def assert_pipeline_status(
             RunningState.FAILED,
             RunningState.ABORTED,
         ]
-
-    MAX_TIMEOUT_S = 60
+    _MINUTE = 60
```

Comment:
MINOR: FYI I started collecting all these constants for tests in ``pytest_simcore.helpers.constants`` so we keep the same names